doc/classes/InputEventKey.xml

```diff
@@ -5,15 +5,42 @@
 	</brief_description>
 	<description>
 		Stores key presses on the keyboard. Supports key presses, key releases and [member echo] events.
+		[b]Note:[/b] Events received from the keyboard usually have all properties set. Event mappings should have only one of the [member keycode], [member physical_keycode] or [member unicode] set.
+		When events are compared, properties are checked in the following priority - [member keycode], [member physical_keycode] and [member unicode], events with the first matching value will be considered equal.
 	</description>
 	<tutorials>
 		<link title="InputEvent">$DOCS_URL/tutorials/inputs/inputevent.html</link>
 	</tutorials>
 	<methods>
+		<method name="as_text_key_label" qualifiers="const">
+			<return type="String" />
+			<description>
+				Returns a [String] representation of the event [member key_label] and modifiers.
+			</description>
+		</method>
+		<method name="as_text_keycode" qualifiers="const">
+			<return type="String" />
+			<description>
+				Returns a [String] representation of the event [member keycode] and modifiers.
+			</description>
+		</method>
+		<method name="as_text_physical_keycode" qualifiers="const">
+			<return type="String" />
+			<description>
+				Returns a [String] representation of the event [member physical_keycode] and modifiers.
+			</description>
+		</method>
+		<method name="get_key_label_with_modifiers" qualifiers="const">
+			<return type="int" enum="Key" />
+			<description>
+				Returns the localized key label combined with modifier keys such as [kbd]Shift[/kbd] or [kbd]Alt[/kbd]. See also [InputEventWithModifiers].
+				To get a human-readable representation of the [InputEventKey] with modifiers, use [code]OS.get_keycode_string(event.get_key_label_with_modifiers())[/code] where [code]event[/code] is the [InputEventKey].
```

Comment:
Not something to change in this PR, but I wonder why this helper is in `OS` and not in `Input`. Would it make sense to move it?